examples/dataflow-streaming-benchmark/README.md

```diff
@@ -71,6 +76,8 @@ Pub/Sub topic.
   "completed": false
 }
 ```
+Since schema includes eventId, eventTimestamp fields, message also contains these fields
+as attributes in addition to regular payload.
```

Comment:
Maybe update to:

```
Since the schema includes the reserved field names of `eventId` and `eventTimestamp`, the output Pub/Sub message will also contain these fields in the message attributes in addition to the regular payload.
```

Reply:
Done